BindFamilyLockBonusActivationActivity/function.json

```diff
@@ -0,0 +1,24 @@
+{
+  "bindings": [
+    {
+      "type": "cosmosDBTrigger",
+      "name": "documents",
+      "direction": "in",
+      "leaseCollectionName": "change-feed-bonus",
+      "leaseCollectionPrefix": "ba",
```

Comment:
```suggestion
      "leaseCollectionPrefix": "babinding",
```

Reply:
fixed in 1c124ec

Reply:
this is needed otherwise the stored checkpoint will overlapthe other function (StoreBonusActivationHistory)